gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -259,22 +302,22 @@ def import_pangolin_vcf(vcf_path: str) -> hl.Table:
         "Number of rows in Pangolin Hail Table after collect_by_key: %s", ht.count()
     )
     ht = ht.select(
-        pangolin=hl.struct(
-            largest_ds=hl.if_else(
-                hl.abs(hl.min(ht.values.largest_ds_gene))
-                > hl.abs(hl.max(ht.values.largest_ds_gene)),
-                hl.min(ht.values.largest_ds_gene),
-                hl.max(ht.values.largest_ds_gene),
-            )
+        pangolin_largest_ds=hl.if_else(
+            hl.abs(hl.min(ht.values.largest_ds_gene))
+            > hl.abs(hl.max(ht.values.largest_ds_gene)),
+            hl.min(ht.values.largest_ds_gene),
+            hl.max(ht.values.largest_ds_gene),
         )
     )
+    # TODO: update version when Invitae publishes new version
```

Comment:
Just confirming they used this new version on the v4 exomes? It seems like we have multiple versions for this tool between the original v3 genomes and v4 exomes, and then the bugfix and the new genomes?

Reply:
They used two versions for v4 genomes, this version for v4 exomes, bug impacted 20M and new genomes variants. I asked Kevin about the first version.